git-tar/function/ops.go

```diff
@@ -181,6 +200,14 @@ func deploy(tars []tarEntry, pushEvent sdk.PushEvent, stack *stack.Services) err
 	c := http.Client{}
 	gatewayURL := os.Getenv("gateway_url")
 
+	event := getEvent(pushEvent)
+	serviceValue := owner + "-" + repoName
+
+	reportStatus("pending", fmt.Sprintf("%s build and deploy is in progress", serviceValue), "DEPLOY", event)
```

Comment:
Is `DEPLOY` our own term or a GitHub constant? Could it be `Deploy`?

Reply:
Its a status context and our own term. We can definitely set it as Deploy

Reply:
It also going to break `of_badge_gen` as the image are mapped by `status + context` [for example:  `success_BUILD.svg`], is it okay to update it in a separate PR?